src/databricks/labs/ucx/framework/owners.py

```diff
@@ -169,8 +171,21 @@ def get_workspace_administrator(self) -> str:
 class Ownership(ABC, Generic[Record]):
     """Determine an owner for a given type of object."""
 
+    _ownerships: set[Ownership] = set()
```

Comment:
I don't think the type annotation here is quite correct: it needs to be `ClassVar[…]`? (Without this I believe it is considered by the type tooling to be an instance property.)